satellite/payments/paymentsconfig/config.go

```diff
@@ -16,10 +16,10 @@ type Config struct {
 	EgressTBPrice            string      `help:"price user should pay for each TB of egress" default:"45"`
 	ObjectPrice              string      `help:"price user should pay for each object stored in network per month" default:"0.0000022"`
 	BonusRate                int64       `help:"amount of percents that user will earn as bonus credits by depositing in STORJ tokens" default:"10"`
-	CouponValue              int64       `help:"coupon value in cents" default:"30"`
+	CouponValue              int64       `help:"coupon value in cents" default:"275"`
```

Comment:
This is the commit message "0.50 for storage, 2.25 for egress" are theses values the values that you wanted to set?

Reply:
That’s the calculation of why the coupon is worth 2.75. 50ct for storing 50GB for one month, 2.25 for downloading 50GB